packages/domain/lib/di/data_model.dart

```diff
@@ -0,0 +1,9 @@
+class GetDataResponse {
+  final Map<String, dynamic> headers;
+  final List<dynamic> body;
+
+  GetDataResponse({
+    required this.headers,
+    required this.body,
+  });
+}
```

Comment:
why is this file in this folder?

Reply:
@egor-slatvinskiy-idf I agree with Liza. why? because in my opinion conditions based on headers should be in the usecase. so we need the model to transfer data from data layer to domain